README.md

```diff
@@ -370,11 +386,11 @@ for i in a_list:
 ''.join(i for in a_list)
 ```
 
-## Collections
+## Collections/Iterables
```

Comment:
You need to update the anchor link to `collectionsiterables` so the link won't break

Reply:
:+1: